packages/@lwc/integration-karma/test/template/attribute-aria/x/parent/parent.html

```diff
@@ -0,0 +1,102 @@
+            aria-colcount="foo"
+            aria-colindex="foo"
+            aria-colspan="foo"
+            aria-controls="foo"
+            aria-current="foo"
+            aria-describedby="foo"
+            aria-details="foo"
+            aria-disabled="foo"
+            aria-errormessage="foo"
+            aria-expanded="foo"
+            aria-flowto="foo"
+            aria-haspopup="foo"
+            aria-hidden="foo"
+            aria-invalid="foo"
+            aria-keyshortcuts="foo"
+            aria-label="foo"
+            aria-labelledby="foo"
+            aria-level="foo"
+            aria-live="foo"
+            aria-modal="foo"
+            aria-multiline="foo"
+            aria-multiselectable="foo"
+            aria-orientation="foo"
+            aria-owns="foo"
+            aria-placeholder="foo"
+            aria-posinset="foo"
+            aria-pressed="foo"
+            aria-readonly="foo"
+            aria-relevant="foo"
+            aria-required="foo"
+            aria-roledescription="foo"
+            aria-rowcount="foo"
+            aria-rowindex="foo"
+            aria-rowspan="foo"
+            aria-selected="foo"
+            aria-setsize="foo"
+            aria-sort="foo"
+            aria-valuemax="foo"
+            aria-valuemin="foo"
+            aria-valuenow="foo"
+            aria-valuetext="foo"
+            role="foo"
+    ></x-child>
+    <div
+            aria-activedescendant="foo"
+            aria-atomic="foo"
+            aria-autocomplete="foo"
+            aria-busy="foo"
+            aria-checked="foo"
+            aria-colcount="foo"
+            aria-colindex="foo"
+            aria-colspan="foo"
+            aria-controls="foo"
+            aria-current="foo"
+            aria-describedby="foo"
+            aria-details="foo"
+            aria-disabled="foo"
+            aria-errormessage="foo"
+            aria-expanded="foo"
+            aria-flowto="foo"
+            aria-haspopup="foo"
+            aria-hidden="foo"
+            aria-invalid="foo"
+            aria-keyshortcuts="foo"
+            aria-label="foo"
+            aria-labelledby="foo"
+            aria-level="foo"
+            aria-live="foo"
+            aria-modal="foo"
+            aria-multiline="foo"
+            aria-multiselectable="foo"
+            aria-orientation="foo"
+            aria-owns="foo"
+            aria-placeholder="foo"
+            aria-posinset="foo"
+            aria-pressed="foo"
+            aria-readonly="foo"
+            aria-relevant="foo"
+            aria-required="foo"
+            aria-roledescription="foo"
+            aria-rowcount="foo"
+            aria-rowindex="foo"
+            aria-rowspan="foo"
+            aria-selected="foo"
+            aria-setsize="foo"
+            aria-sort="foo"
+            aria-valuemax="foo"
+            aria-valuemin="foo"
+            aria-valuenow="foo"
+            aria-valuetext="foo"
+            role="foo"
+    ></div>
```

Comment:
Note that the LWC compiler emits different output for ARIA attrs on LWC elements rather than generic HTML elements. (It uses the prop format for one and the attr format for the other.) Hence I'm testing both `<x-child>` and a `<div>`.

Reply:
Could we add a comment in `aria.ts` that links back to this section?  In case the list changes in `aria.ts` there's a note to update this section as well.